source/threads.tex

```diff
@@ -9256,6 +9256,7 @@
 \rSec2[semaphore.syn]{Header \tcode{<semaphore>} synopsis}
 
 \indexheader{semaphore}%
+\indexlibraryglobal{binary_semaphore}%
```

Comment:
Please move to the place where it is defined (a few lines below) and use  @\libglobal{blah}@

That way, we point to the right spot within the synopsis, not at its beginning.